SonarQube.Client/Models/ServerSentEvents/ClientContract/ITaintIssue.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+
+namespace SonarQube.Client.Models.ServerSentEvents.ClientContract
+{
+    public interface ITaintIssue
+    {
+        //DateTimeOffset CreationDate { get; }
+        string RuleKey { get; }
+        SonarQubeIssueSeverity Severity { get; }
+        SonarQubeIssueType Type { get; }
+        ILocation MainLocation { get; }
+        IFlow[] Flows { get; }
+    }
+
+    public interface IFlow
+    {
+        ILocation[] Locations { get; }
+    }
+
+    public interface ILocation
+    {
+        string FilePath { get; }
+        string Message { get; }
+        ITextRange TextRange { get; }
+    }
+
+    public interface ITextRange
+    {
+        int StartLine { get; }
+        int StartLineOffset { get; }
+        int EndLine { get; }
+        int EndLineOffset { get; }
+        string Hash { get; }
+    }
+
+    internal class TaintIssue : ITaintIssue
```

Comment:
Sadly the old api returns SonarQubeIssue and the new API for taint returns this, and they're similar but not the same.

Reply:
Is this a big or a small issue? I don't have the needed context to understand this :-(

Reply:
I was hoping that I could simply use the existing model SonarQubeIssue but sadly I can't, so I had to create a new one